core/src/main/scala/libretto/scaletto/StarterAppBase.scala

```diff
@@ -2,4 +2,5 @@ package libretto.scaletto
 
 abstract class StarterAppBase {
   export StarterKit._
+  export StarterKit.scalettoLib.given
```

Comment:
I thing something like that is also needed one level deeper.

See below for more…